config/zones.json

```diff
@@ -1,4 +1,11 @@
 {
+  "AU": {},
```

Comment:
Are these needed? If not, then I suggest not adding them to the config as it's used throughout the systems and might have unforeseen consequences

Reply:
Good spotting, it's an artefact from before I knew we explicitly defined sub-zones. I shall remove it.